model/model.js

```diff
@@ -59,15 +59,24 @@ exports.selectArticles = () => {
     })
 }
 
-exports.selectArticlesComments = (id) => {
+exports.selectArticlesComments = async (id) => {
+    const articles = await db.query(`SELECT * FROM articles`)
     return db.query(`SELECT * FROM comments WHERE article_id = $1`,[id])
     .then((result) => {
+        const numArticles = articles.rows.length;
         result.rows[0];
-        if (!result.rows[0]){ 
-            return Promise.reject({
-                status: 404,
-                msg: 'No article found with that id'
-            })
+        if (!result.rows[0]){
+            if (id <= numArticles){
+                return Promise.reject({
+                    status: 200,
+                    msg: 'No comments for this article'
```

Comment:
This shouldnt be a rejection as technically there is no error here and you are sending a 200. From a users standpoint, they are expecting an array of comments. If there is no comments, we should be sending back an empty array for data consistency.

We're going to have to check if the article exists as well as checking for any comments